### PR TITLE
BENCH-262 home page

### PR DESCRIPTION
**Pull request by AudriusPutys**

### CODE REVIEW

**Comment by AudriusPutys** [comment]
image

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        </a>
+        <a className="cursor-pointer hover:text-white" onClick={() => navigate(RouteConstants.POLICIES)}>
+          View Policies
+        </a>
+      </div>
+    </>
+  );
+};
+
+export default Footer;
```
Comment (harrystead2) [inline src/components/Footer/Footer.tsx]: There's two exports here, just stick with the first export and remove this one

**Comment by harrystead2** [comment]
Looks great!

File: src/components/Footer/Footer.tsx (hunk 1 above)
Comment (AudriusPutys) [reply src/components/Footer/Footer.tsx]: removed it from Footer and HomePage

**Review by harrystead2 — APPROVED** [review]

**Review by bethcryer — APPROVED** [review]

**Pull request merged**